stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -108,7 +108,7 @@ extension TensorHandle : TensorSendableReceivable {
     TF_DeleteTensor(cTensor!)
     if _RuntimeConfig.printsDebugLog {
       debugLog("The received tensor of id \(tensorId) has content:")
-      dumpTensorContent(tensorHandle.cTensorHandle, Scalar.self)
+      // dumpTensorContent(tensorHandle.cTensorHandle, Scalar.self)
```

Comment:
To properly support `dumpTensorContents`, you should make `TensorHandle` **conditionally** conform to `TensorSendableReceivable` where `Scalar` conforms to `AccelerableByTensorFlow` on line 93. After doing that, you should be able to uncomment these lines.